src/main/scala/com/databricks/spark/avro/SchemaConverters.scala

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2014 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.databricks.spark.avro
+
+import scala.collection.JavaConversions._
+
+import org.apache.avro.{Schema, SchemaBuilder}
+import org.apache.avro.SchemaBuilder._
+import org.apache.spark.sql._
+import org.apache.spark.sql.catalyst.types.{MapType, ArrayType}
+import org.apache.avro.Schema.Type._
+
+
+/**
+ * This object contains method that are used to convert sparkSQL schemas to avro schemas and vice
+ * versa.
+ */
+private object SchemaConverters {
```

Comment:
Note that you can always make things `private[avro]` so that you can access them internally but not expose them to users (e.g., through javadoc)